cli/src/pcluster/api/controllers/image_operations_controller.py

```diff
@@ -206,7 +206,7 @@ def describe_image(image_id, region=None):
 def _image_to_describe_image_response(imagebuilder):
     return DescribeImageResponseContent(
         creation_time=to_iso_time(imagebuilder.image.creation_date),
-        image_configuration=ImageConfigurationStructure(url=imagebuilder.config_url),
+        image_configuration=ImageConfigurationStructure(url=imagebuilder.presigned_config_url),
```

Comment:
we should not fail if the s3 bucket or config file cannot be found. 

In the cluster controller we are returning a NOT_AVAILABLE string
```
    config_url = "NOT_AVAILABLE"
    try:
        config_url = cluster.config_presigned_url
    except ClusterActionError as e:
        # Do not fail request when S3 bucket is not available
        LOGGER.error(e)
```